tests/hardening/5.2.7_sshd_maxauthtries.sh

```diff
@@ -7,6 +7,13 @@ test_audit() {
     # shellcheck disable=2154
     run blank /opt/debian-cis/bin/hardening/"${script}".sh --audit-all
 
+    echo "MaxAuthTries 6" >>/etc/ssh/sshd_config
+
+    describe Running too permissive
+    register_test retvalshouldbe 1
+    register_test contain "[ KO ] 6 is higher than recommended 4"
+    run permissive /opt/debian-cis/bin/hardening/"${script}".sh --audit-all
+
```

Comment:
Blocker : Please add a MaxAuthTries lower than recommended value and check that the check passes.